### ISSUE TITLE
 Initial time, Increment min. max. Elo

### ISSUE THREAD

**Issue by antiproton** [jsonchess/lightsquare#117]
it is possible to change the initial time/ Increment/ min. max. Elo.
example: http://netreal.de/test1
Best regards R.U.

**Comment by gushogg-blake** [comment]
Yeah I think sliders are better for these kinds of input.  The "range" input type is now supported in most recent browsers, so I'll experiment with using that before bringing jQuery in.  I think we need to display the actual value somewhere as well, and maybe let the user edit that in a normal text box if they want

**Comment by gushogg-blake** [comment]
Also we need users to be able to create games from 30 seconds initial time, up to at least 1 day.  So the steps should go like 30s, 1m, 2m, 3m, 5m, 10m, 20m, 30m, 60m, etc.  Will be a fair bit of work, but probably worth it.

**Comment by antiproton** [comment]
A simple and quick solution!
example: http://netreal.de/test5/test.htm

**Comment by gushogg-blake** [comment]
I like the sliders though :)
